pkg/command/context.go

```diff
@@ -131,16 +131,16 @@ var createCtxCmd = &cobra.Command{
     # Create a TKG management cluster context using default kubeconfig path and a kubeconfig context
     tanzu context create mgmt-cluster --kubecontext kubecontext
 
-    # Create a Application Engine(TAE) context with default endpoint (--type option is not necessary for default endpoint)
-    tanzu context create mytae -endpoint https://api.tanzu.cloud.vmware.com 
+    # Create an Application Engine (TAE) context with the default endpoint (--type is not necessary for the default endpoint)
+    tanzu context create mytae --endpoint https://api.tanzu.cloud.vmware.com 
```

Comment:
nit: there is a trailing space here

Reply:
Thanks for noticing @vuil .  I have opened #522